Expo1900/Expo1900/View/KoreaExhibitViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  KoreaExhibitItemController.swift
+//  Expo1900
+//
+//  Created by sookim on 2021/04/09.
+//
+
+import UIKit
+
+class KoreaExhibitViewController: UIViewController {
+    
+    @IBOutlet weak var koreaExhibitTableView: UITableView!
+    
+    private var koreaExhibitItems: [KoreaExhibitItem] = []
+
+    
+    override func viewDidLoad() {
+        title = "한국의 출품작"
+        
+        koreaExhibitTableView.delegate = self
+        koreaExhibitTableView.dataSource = self
+        itemDecode()
+    }
+    
+    private func itemDecode() {
+        let jsonDecoder = JSONDecoder()
+        guard let koreaExhibitItemsData = NSDataAsset(name: "items")
+        else { return }
```

Comment:
guard에서 바로 return 처리하는 경우에는 보통 다음과 같이 표기를 많이 합니다! 참고하세요 😀
```suggestion
        guard let koreaExhibitItemsData = NSDataAsset(name: "items")  else { return }
```

Reply:
guard문 표기법 주의해야겠네요😅   덕분에 깃허브 suggestion 기능에 대해서 배웠어요 ㅎㅎ